.github/dependabot.yaml

```diff
@@ -9,6 +9,6 @@ updates:
     labels:
       - autosubmit
     groups:
-      github-actions:
+      dependencies:
```

Comment:
Is this part of an automated change? We want `github-actions` here; I'll check my disk to see that I don't have anything uncommitted.

Reply:
I just ran the robot from latest... 🤷